app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -83,6 +83,11 @@ class FxaRepo(
         accountManager.initAsync() // If user is already logged in, the appropriate observers will be triggered.
     }
 
+    fun logout() {
+        @Suppress("DeferredResultUnused") // No value is returned & we don't need to wait for this to complete.
+        accountManager.logoutAsync()
```

Comment:
nit: while you're in here, can you make the accountManager private? I don't think any consumers need it.

Reply:
It's used by the login use case, unfortunately.  I __really__ miss package private visibility.

Reply:
We could totally wrap that LoginUseCase method in another function to avoid exposing the `AccountManager` in its entirety. Maybe we'll fix that the next time we're in this code. :)